msteams-platform/messaging-extensions/high-quality-message-extension.md

```diff
@@ -158,6 +158,8 @@ Command description is used to map user intent and utterance to search command i
 * Include verbs and synonyms, if applicable.
 * Focus on keywords that are likely to be used in the search function of your native apps.
 
+Use the [semanticDescription](../resources/schema/manifest-schema-dev-preview.md#composeextensionscommands) field to describe your command for consumption by Microsoft Copilot, including expected values, limits, and ranges. Unlike the regular `description` field, *semanticDescription* allows for up to 5000 characters and will never be exposed in the user interface. If left empty, Copilot will fall back to the *description* field.
```

Comment:
```suggestion
 The [semanticDescription](../resources/schema/manifest-schema-dev-preview.md#composeextensionscommands) property is used to provide a detailed description of a command for Microsoft Copilot. This field allows for up to 5000 characters and isn't displayed in the user interface. If the `semanticDescription` property is left empty, Copilot uses the information in the `description` field. When writing a `semanticDescription`, you must include information about expected values, limits, and ranges for the command.
```

Reply:
Saying "you must include" might be too strong, as I don't think this is enforced as a requirement. How about:

> When writing a `semanticDescription`, include information about expected values, limits, and ranges for the command.

Reply:
I thought that this is a mandatory requirement to add semantic description hence I used the term "you must" to emphasize. Please correct me if my understanding is not correct.